src/Integration.Vsix/CFamily/VcxProject/CmdBuilder.cs

```diff
@@ -63,9 +72,7 @@ public void AddCompiler(string path)
 
         internal /* for testing */ static string AddQuote(string s)
         {
-            // To avoid paths with spaces
-            Regex doubleQuoted = new Regex("\"[^\\\"]*(\\.[^\\\"]*)*\"");
```

Comment:
This method and the next one are called multiple times for every C++ analysis. There's no reason to create a new `Regex` each time, so I moved the definitions to `static readonly` constants.

The unit test coverage of these two methods is very good.